package.json

```diff
@@ -104,8 +105,8 @@
     "circleci:lint": "npm run lint-files && npm run nsp",
     "build:examples": "cd examples && node buildAll.js",
     "pretest": "npm run lint-files",
-    "lint-files": "npm run lint && npm run beautify-lint && npm run schema-lint",
-    "lint": "eslint lib bin hot buildin \"test/*.js\" \"test/**/webpack.config.js\" \"test/binCases/**/test.js\" \"examples/**/webpack.config.js\"",
+    "lint-files": "npm run lint",
+    "lint": "eslint lib bin hot buildin \"test/*.test.js\" \"test/**/webpack.config.js\" \"test/binCases/**/test.js\" \"examples/**/webpack.config.js\"",
     "fix": "npm run lint -- --fix",
```

Comment:
Since Prettier supports Markdown, JSON, CSS and more, we could add one more rule to fix those as well

```
"fix:non-js": "prettier --ignore-path .gitignore --write \"**/*.{json,css,md}\"",
```

and it would be a good idea to add `.prettierignore` file to exclude the files/paths that we don't want to touch.